test/puppetlabs/puppetdb/http/metrics_test.clj

```diff
@@ -37,22 +33,23 @@
   [req]
   (assoc-in req [:headers "accept"] "text/plain"))
 
-(deftestseq metrics-set-handler
+(let [app (server/build-app)]
+  (deftestseq metrics-set-handler
```

Comment:
I guess that's fine -- I was more wondering if build-app could be folded into the existing function in server.clj to save a function, but if we're splitting metrics keeping them separate may make more sense.